.travis.yml

```diff
@@ -14,7 +14,9 @@ before_script:
   - travis_retry composer self-update
   - travis_retry composer update --no-interaction --prefer-source
 
-script: make test-coverage-clover
+script:
```

Comment:
I would like to make this all dockerised. I did it with graze/array-filter https://github.com/graze/array-filter/blob/master/.travis.yml but it doesn't handle different php versions currently

Do you think there is a way to use travis different php versioning and allow failures with the matrix?